File: src/main/java/com/projectswg/forwarder/services/crash/CrashManager.java
package com.projectswg.forwarder.services.crash;

import me.joshlarson.jlcommon.control.Manager;
import me.joshlarson.jlcommon.control.ManagerStructure;

@ManagerStructure(children = {
		PacketRecordingService.class,
		IntentRecordingService.class
})
public class CrashManager extends Manager {
	
}

File: src/main/kotlin/com/projectswg/forwarder/services/crash/CrashManager.kt
package com.projectswg.forwarder.services.crash

import me.joshlarson.jlcommon.control.Manager
import me.joshlarson.jlcommon.control.ManagerStructure

@ManagerStructure(children = [
	PacketRecordingService::class,
	IntentRecordingService::class
])
class CrashManager : Manager()
